modules/p2/ting_openlist/include/ting_openlist_analysis.inc

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * @file
+ * Functions for analysis of local OpenList Entities
+ */
+
+function ting_openlist_get_local_analysis() {
+  $data = ting_openlist_get_local_analysis_data();
+  return '<pre>' . htmlentities(var_export($data, TRUE)) . '</pre>';
+}
+
+function ting_openlist_submit_local_analysis() {
+  dpm(ding_provider_invoke('openlist', 'call_module', 'LocalAnalysis', 'sendData', array(ting_openlist_get_local_analysis_data())));
+
+
+  return "Stats are sent.";
+}
+
+function ting_openlist_get_local_analysis_data() {
```

Comment:
Please add documentation of what this function does.

The result of this function is passed directly to OpenList for analysis so it is important that the structure of the return value is described.